trojan.dmitry/task-2-1/cmd/service/main.go

```diff
@@ -0,0 +1,83 @@
+package main
+
+import (
+	"fmt"
+)
+
+type Dept struct {
+	minLevel int
+	maxLevel int
+}
+
+func NewDept() Dept {
+	const (
+		defaultMinLevel = 15
+		defaultMaxLevel = 30
+	)
+
+	return Dept{minLevel: defaultMinLevel, maxLevel: defaultMaxLevel}
+}
+
+func (department *Dept) Update(operator string, num int) {
+	switch operator {
+	case ">=":
+		department.minLevel = max(department.minLevel, num)
+	case "<=":
+		department.maxLevel = min(department.maxLevel, num)
+	}
+}
+
+func (department *Dept) Result() int {
+	if department.minLevel <= department.maxLevel {
+		return department.minLevel
+	}
+
+	return -1
+}
+
+func main() {
+	var (
+		department int
+		workers    int
+		num        int
+		operator   string
+	)
+
+	_, err := fmt.Scan(&department)
+	if err != nil {
+		fmt.Println("Invalid number of departments")
+
+		return
+	}
+
+	for range department {
+		_, err = fmt.Scan(&workers)
+		if err != nil {
+			fmt.Println("Invalid number of workers")
```

Comment:
```suggestion
			fmt.Println("Invalid number of workers", err)
```

там ведь много всего полезного